invoice/client_test.go

```diff
@@ -91,6 +96,14 @@ func TestAllInvoicesScenarios(t *testing.T) {
 		t.Errorf("Invoice tax  %v does not match expected tax 20\n", targetInvoice.Tax)
 	}
 
+	if targetInvoice.DueDate != dueDate {
+		t.Errorf("Invoice days until due %v does not match expected %v\n", targetInvoice.DueDate, dueDate)
+	}
+
+	if targetInvoice.Billing != invoiceParams.Billing {
+		t.Errorf("Invoice billing %v does not match expected send invoice\n", targetInvoice.Billing)
```

Comment:
nit, did you mean `does not match expected send_invoice` or format the string with another`%v` and pass in  `invoiceParams.Billing`?